mimic/model/nova_objects.py

```diff
@@ -527,6 +527,35 @@ def fail_later(server):
     return fail_later
 
 
+@server_creation.declare_behavior_creator("sequence")
+def sequence(parameters):
+    """
+    Sometimes a sequence of behaviors occur when you try to create a server in
+    a predictable pattern.
+
+    Takes one parameter, ``behaviors``, which is a list of specifications of
```

Comment:
This … isn't quite true? Normally they take a name under "name" and parameters under "parameters" and criteria under "criteria"; here, the criteria are already specified.  Perhaps an example dictionary would be helpful though.